harmonica/_transformations.py

```diff
@@ -393,6 +393,69 @@ def total_gradient_amplitude(grid):
     return np.sqrt(gradient[0] ** 2 + gradient[1] ** 2 + gradient[2] ** 2)
 
 
+def tilt(grid):
+    r"""
+    Calculates the tilt of a potential field grid as defined
+    by Miller and Singh (1994)
```

Comment:
The first line should fit in 79 chars. You can leave the citation to the rest of the docstring.

Reply:
A suggestion would be `Calculates the tilt angle of a potential field grid`